rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/IGraphConfigurationService.java

```diff
@@ -43,6 +43,11 @@
     Schema getSchema();
 
     @GET
+    @Path("/description")
+    @ApiOperation(value = "Gets the Graph description", response = String.class, responseContainer = "list")
```

Comment:
There shouldn't be a responseContainer parameter here, since this operation returns a single value and not a list.